tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu

```diff
@@ -125,7 +125,6 @@ GeneratedTests.select_v1_2_one_dim_quant8
 GeneratedTests.select_v1_2_two_dim_quant8
 GeneratedTests.slice_5
 GeneratedTests.slice_6
-GeneratedTests.slice_7
```

Comment:
Why only slice_7 is enabled?

Reply:
5,6,8 seems to int32 tests.
but we don't have int kernel yet.
so I think it is fine to disable 5,6,8.

Reply:
@btheosam 
Refer to `slice.mod.py`. There is eight of testcase.
The 7th is about quant8.